README.md

```diff
@@ -45,16 +32,33 @@ L:   1 | P:  14 | L006 | Operators should be surrounded by a single space unless
 L:   1 | P:  27 | L001 | Unnecessary trailing whitespace
 ```
 
-# Usage
+You can also have a play using [SQLFluff online](https://sqlfluff-online.herokuapp.com/).
+
+For full CLI usage, see the docs.
+
+# Documentation
 
-For more details on usage see the docs on readthedocs [here](http://sqlfluff.readthedocs.io).
-You can also explore fluff using the [SQLFluff online](https://sqlfluff-online.herokuapp.com/) website.
+For full documentation visit [SQLFluff on readthedocs.io](http://sqlfluff.readthedocs.io).
```

Comment:
Canonical URL is `docs.sqlfluff.com`, the url you've put in will just redirect to that.

Reply:
Cool thanks, updated